docs/standard/library.md

```diff
@@ -1,105 +1,124 @@
+The various .NET runtimes implement specific versions of .NET Standard. Each .NET runtime version advertises the highest .NET Standard version it supports, a statement that means it also supports previous versions. For example, the .NET Framework 4.6 implements .NET Standard 1.3, which means that it exposes all APIs defined in .NET Standard versions 1.0 through 1.3. Similarly, the .NET Framework 4.6.1 implements .NET Standard 1.5, while .NET Core 1.0 implements .NET Standard 1.6.
 
-## .NET Platforms Support
+## .NET platforms support
 
-You can see the complete set of .NET runtimes that support the .NET Standard Library.
+The following table lists all versions of .NET Standard and the platforms supported:
 
-| Platform Name | Alias |  |  |  |  |  | | | |
-| :---------- | :--------- |:--------- |:--------- |:--------- |:--------- |:--------- |:--------- |:--------- |:--------- |
-|.NET Standard | netstandard | 1.0 | 1.1 | 1.2 | 1.3 | 1.4 | 1.5 | 1.6 | 2.0 |
-|.NET Core|netcoreapp|&rarr;|&rarr;|&rarr;|&rarr;|&rarr;|&rarr;|1.0|2.0|
-|.NET Framework|net|&rarr;|4.5|4.5.1|4.6|4.6.1|4.6.2|vNext|4.6.1|
-|Mono/Xamarin Platforms||&rarr;|&rarr;|&rarr;|&rarr;|&rarr;|&rarr;|&rarr;|vNext|
-|Universal Windows Platform|uap|&rarr;|&rarr;|&rarr;|&rarr;|10.0|&rarr;|&rarr;|vNext|
-|Windows|win|&rarr;|8.0|8.1||||||
-|Windows Phone|wpa|&rarr;|&rarr;|8.1||||||
-|Windows Phone Silverlight|wp|8.0||||||||
+[!INCLUDE [net-standard-table](../includes/net-standard-table.md)]
+
+- The columns represent .NET Standard versions. Each header cell is a link to a document that shows which APIs got added in that version of .NET Standard.
+- The rows represent the different .NET platforms.
+- The version number in each cell indicates the *minimum* version of the platform you'll need to implement that .NET Standard version.
+
+To find the highest version of .NET Standard that you can target, do the following:
+1. Find the row that indicate the platform you want to run on.
+2. Find the column in that row that indicates your version starting from right to left.
+3. The column header indicates the .NET Standard version that supports your target (and any lower .NET Standard versions will also support it).
+4. Repeat this process for each platform you want to target. If you have more than one target platform, you should pick the common version among them, which would be the smaller version. For example, if you want to run on .NET Framework 4.5 and .NET Core 1.0, the highest .NET Standard version you can use is .NET Standard 1.1.
+
```

Comment:
@weshaggard I put everything that was part of how to read the table. The information on how to find the highest I left it out, since the other topic that uses this has its own info about what to target, etc.